### ISSUE TITLE
gravityview/fields/fileupload/link_atts filter doesn't work

### ISSUE THREAD

**Issue by rafaehlers** [gravityview/GravityView#1305]
I can't use this filter to add a CSS class to an image uploaded with a File Upload field:

```// GravityView: Override the default lightbox by FooBox
add_filter( 'gravityview/fields/fileupload/link_atts', 'my_gv_image_link_atts', 10, 2 );
function my_gv_image_link_atts( $link_atts, $field ) {     
    $link_atts['class'] = 'foobox';
    return $link_atts;
}
```

![image](https://user-images.githubusercontent.com/1222472/56508037-8c00ae80-64f9-11e9-9461-547a76c458b3.png)

This prevents the user from using FooBoox as an alternative lightbox plugin.

Tested on GV 2.3.1

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/995529792029955/1119515728496590)

**Comment by rafaehlers** [comment]
https://secure.helpscout.net/conversation/831286013/20291

**Comment by soulseekah** [comment]
The filter was only called if the "Show as Link" setting was on. Fixed.

**Comment by rafaehlers** [comment]
>The filter was only called if the "Show as Link" setting was on. Fixed. Please test.

@soulseekah that's still the case though.

**Comment by soulseekah** [comment]
@rafaehlers are you sure you have turned off secure download links?

It works well for me with:

```php
add_filter( 'gravityview/fields/fileupload/allow_insecure_lightbox', '__return_true' );
```

Remember, no lightboxes work if download links are secured for the field.

Or does foobox work with secure links?

**Comment by zackkatz** [comment]
This will be resolved with #1386 and helped by #1385 

I'm closing because it works, but it ain't great.
